Update PR template (#7794)
TheAlgorithms/Python: .github/pull_request_template.md

* [ ] This pull request is all my own work -- I have not plagiarized.
* [ ] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [ ] All functions and variable names follow Python naming conventions.
* [ ] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.

> * [ ] All new algorithms have a URL in their comments that point to Wikipedia or another similar explanation.

Reviewer: Perhaps this
```suggestion
* [ ] All new algorithms have a docstring containing a URL that point to Wikipedia or another similar explanation.
```
or 
```suggestion
* [ ] All new algorithms include at least one URL that points to Wikipedia or another similar explanation.
```
These make a bit more sense in my opinion
